### PR TITLE
feat: update js-reader to enable DCE

### PR DESCRIPTION
**Pull request by digitalsadhu**
## JIRA Issue
[COREWEB-196](https://jira.finn.no/browse/COREWEB-196)

## Status
**READY**

## Description
Adds NODE_ENV environment variable replacement which unlocks dead code elimination achievement

## Todos
- [x] Tests

## Deploy Notes
* run integration tests
* update and publish new version of podium-asset-server
* update and publish podium podlet with new version of @asset-pipe/client
* update all podlets and layouts with new version of podium podlet
* deploy podium-asset-server and all podlets and layouts

## Related PRs
* https://github.com/asset-pipe/integration/pull/39

### CODE REVIEW

File: lib/main.js [hunk 1]
```diff
@@ -205,6 +205,7 @@ module.exports = class Router extends EventEmitter {
                     type,
                     sourceMaps,
                     minify,
+                    env: process.env.NODE_ENV,
```
Comment (SimenB) [inline lib/main.js]: it would be great if this would be an option to the constructor

Comment (digitalsadhu) [reply lib/main.js]: 'tis but the work of a moment

Comment (digitalsadhu) [reply lib/main.js]: I shall dip it in yoghurt and cover it with chocolate buttons

Comment (SimenB) [reply lib/main.js]: 🍫

**Pull request merged**